src/Tools/dotnet-trace/CommandLine/Commands/CollectLinuxCommand.cs

```diff
@@ -47,9 +53,11 @@ public CollectLinuxCommandHandler(IConsole console = null)
         /// </summary>
         internal int CollectLinux(CollectLinuxArgs args)
         {
-            if (!OperatingSystem.IsLinux())
```

Comment:
GetRid is nothing I wouldn't want to surface - even internally. I'd expose instead an "is supported" method or just inline it here.

Reply:
I grabbed it from here https://github.com/dotnet/diagnostics/blob/8d3bda12bfaf93adb81d448bdf078d8039f2b479/src/SOS/SOS.InstallHelper/InstallHelper.cs#L313-L345

Is surfacing it there different from here?